### PR TITLE
12.0 mig capital subscription

### PR DESCRIPTION
**Pull request by ajay-odoo**
Migrate capital_subscription module from Version 9 to Version 12

### CODE REVIEW

File: capital_subscription/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, exceptions, _
+from odoo.exceptions import Warning
```
Comment (ivantodorovich) [inline capital_subscription/models/account_invoice.py]: `Warning` was deprecated, please replace with `UserError`

https://github.com/odoo/odoo/blob/12.0/odoo/exceptions.py#L36

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Hi Ajay!

Looks really good! Thanks!

Please take a look at the following comments:

- Check the following PEP8 errors given by travis
```
capital_subscription/models/account_invoice_refund.py:29:77: E502 the backslash is redundant between brackets
capital_subscription/models/account_invoice_refund.py:62:1: W391 blank line at end of file
capital_subscription/models/account_invoice.py:121:80: E501 line too long (80 > 79 characters)
capital_subscription/models/capital_fundraising_category.py:6:1: F401 'datetime.datetime' imported but unused
capital_subscription/models/capital_fundraising_category.py:7:1: F401 'pytz' imported but unused
capital_subscription/tests/common.py:16:80: E501 line too long (82 > 79 characters)
capital_subscription/tests/common.py:17:80: E501 line too long (107 > 79 characters)
capital_subscription/tests/common.py:19:80: E501 line too long (99 > 79 characters)
capital_subscription/tests/common.py:20:80: E501 line too long (98 > 79 characters)
capital_subscription/tests/test_fund.py:3:1: E302 expected 2 blank lines, found 1
capital_subscription/tests/test_fund.py:6:80: E501 line too long (97 > 79 characters)
capital_subscription/tests/test_fund.py:18:1: W293 blank line contains whitespace
capital_subscription/tests/test_fund.py:20:80: E501 line too long (84 > 79 characters)
capital_subscription/tests/test_fund.py:28:80: E501 line too long (97 > 79 characters)
capital_subscription/tests/test_fund.py:42:80: E501 line too long (84 > 79 characters)
```

And also these ones:

In fact there are a few warning indicators here, like the duplicated id
```
************* Module capital_subscription
capital_subscription/demo/account_journal.xml:10: [W7902(duplicate-xml-record-id), ]  Duplicate xml record id "demo/capital_journal_noupdate_0" in demo/account_journal.xml:17
capital_subscription/security/ir.model.access.csv:1: [W7906(duplicate-id-csv), ]  Duplicate id ""
capital_subscription/data/data_product_product.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline
capital_subscription/views/view_account_invoice.xml:19: [W7910(wrong-tabs-instead-of-spaces), ]  Use wrong tabs indentation instead of four spaces

************* Module capital_subscription.models.account_invoice_refund
capital_subscription/models/account_invoice_refund.py:3: [W0622(redefined-builtin), ] Redefining built-in 'Warning'
capital_subscription/models/account_invoice_refund.py:46: [C8107(translation-required), AccountInvoiceRefund._check_refund_quantity_is_positive] String parameter on "Warning" requires translation. Use _('Error! The refund quantity must be greater than 0.')
************* Module capital_subscription.models.account_invoice
capital_subscription/models/account_invoice.py:6: [W0622(redefined-builtin), ] Redefining built-in 'Warning'
************* Module capital_subscription.models.res_partner
capital_subscription/models/res_partner.py:18: [C8108(method-compute), ResPartner] Name of compute method should start with "_compute_"
```

-----

btw we don't care about these two for now. I'll see if I remove it from the tests
```
************* Module capital_subscription.__manifest__
capital_subscription/__manifest__.py:7: [C8101(manifest-required-author), ] One of the following authors must be present in manifest: 'Odoo Community Association (OCA)'
capital_subscription/__manifest__.py:7: [C8102(manifest-required-key), ] Missing required key "license" in manifest file
```

----


Also, test are passing but there were a few WARNINGS that might indicate something is not working as expected

Can you take a look into that?

```
2019-09-13 04:21:07,330 5927 WARNING openerp_test odoo.models: product.category.create() with unknown fields: type
```
```
2019-09-13 04:21:08,702 5927 WARNING openerp_test odoo.models: account.move.create() with unknown fields: payment_id
```

**Comment by ivantodorovich** [comment]
```
capital_subscription/__manifest__.py:7: [C8102(manifest-required-key), ] Missing required key "license" in manifest file
```

btw just talked to @msdruidoo and if the 'licence' is missing, we'll add `AGPL-3`

**Comment by ajay-odoo** [comment]
> ```
> capital_subscription/__manifest__.py:7: [C8102(manifest-required-key), ] Missing required key "license" in manifest file
> ```
> 
> btw just talked to @msdruidoo and if the 'licence' is missing, we'll add `AGPL-3`

**Pull request merged**